.infrastructure/cronjob.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: health-monitor
+  namespace: mateapp
+spec:
+  schedule: "*/4 * * * *"
+  concurrencyPolicy: Allow
```

Comment:
The `concurrencyPolicy` value should be lowercase. Change `Allow` to `allow` to match the expected values (`Allow`, `Forbid`, `Replace`).